src/sinks/prometheus/remote_write.rs

```diff
@@ -57,6 +57,9 @@ enum Errors {
 #[serde(deny_unknown_fields)]
 pub struct RemoteWriteConfig {
     /// The endpoint to send data to.
+    ///
+    /// The endpoint should include the scheme and the path to write to.
+    #[configurable(metadata(docs::examples = "https://localhost:8087/"))]
```

Comment:
**suggestion**: use `/api/v1/write` in an example.

This is the default path when enabling remote write on a Prometheus server.